### PR TITLE
[12.0][MIG] account_invoice_analytic_search

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Standard migration

### CODE REVIEW

File: account_invoice_analytic_search/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Invoice Analytic Search",
+    "summary": "Search invoices by analytic account or by project manager",
+    "version": "12.0.1.0.0",
+    "author": "Project Expert Team,"
+    "Eficent,"
+    "Odoo Community Association (OCA)",
+    "website": "http://project.expert",
```
Comment (nikul-serpentcs) [inline account_invoice_analytic_search/__manifest__.py]: https://github.com/OCA/account-invoicing

File: account_invoice_analytic_search/models/account_invoice.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```
Comment (nikul-serpentcs) [inline account_invoice_analytic_search/models/account_invoice.py]: @aheficent not needed

File: account_invoice_analytic_search/README.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+project or to an analytic account, searching by it's code or name.
+This module introduces the possibility search customer or supplier
+invoices by analytic account.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/95/10.0
```
Comment (nikul-serpentcs) [inline account_invoice_analytic_search/README.rst]: 12.0

File: account_invoice_analytic_search/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```
Comment (nikul-serpentcs) [inline account_invoice_analytic_search/__manifest__.py]: same here

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Minor change

**Comment by AaronHForgeFlow** [comment]
Comments attended @nikul-serpentcs  Thanks for reviewing! :)

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Comment by AaronHForgeFlow** [comment]
@nikul-serpentcs could you please update review bro?

**Comment by AaronHForgeFlow** [comment]
Ping @nikul-serpentcs

File: account_invoice_analytic_search/tests/test_account_invoice_analytic_search.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+            {"name": "Test Account", "code": "TA"}
+        )
+        self.invoice = self.account_invoice.create(
+            {
+                "partner_id": self.partner_2,
+                "account_id": self.invoice_account,
+                "type": "in_invoice",
+            }
+        )
+        self.env["account.invoice.line"].create(
```
Comment (nikul-serpentcs) [inline account_invoice_analytic_search/tests/test_account_invoice_analytic_search.py]: self.env["account.invoice.line"] -> self.account_invoice_line

File: account_invoice_analytic_search/tests/test_account_invoice_analytic_search.py [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+                "product_id": self.product_4,
+                "quantity": 1.0,
+                "price_unit": 100.0,
+                "invoice_id": self.invoice.id,
+                "name": "product that cost 100",
+                "account_id": self.invoice_line_account,
+                "account_analytic_id": self.analytic_account.id,
+            }
+        )
+        self.env["account.invoice.line"].create(
```
Comment (nikul-serpentcs) [inline account_invoice_analytic_search/tests/test_account_invoice_analytic_search.py]: same here

**Review by nikul-serpentcs — APPROVED** [review]
Minor Change, Otherwise Code Review LGTM :+1:

File: account_invoice_analytic_search/tests/test_account_invoice_analytic_search.py (hunk 5 above)
Comment (AaronHForgeFlow) [reply account_invoice_analytic_search/tests/test_account_invoice_analytic_search.py]: Comments attended. Thanks!

**Comment by AaronHForgeFlow** [comment]
squashing bot commits, ready to merge.

**Pull request merged**